rust-connectors/common/src/config.rs

```diff
@@ -89,8 +90,39 @@ impl ConnectorConfig {
                 producer.batch_size = Some(batch_size);
             }
         }
+        debug!("Using connector config {connector_config:#?}");
         Ok(connector_config)
     }
+
+    pub fn consumer_parameters(&self) -> Vec<String> {
+        let mut params = Vec::new();
+        if let Some(consumer) = self.consumer.as_ref() {
+            if let Some(partition) = consumer.partition {
+                params.push("--consumer-partition".to_string());
+                params.push(format!("{}", partition));
+            }
+        }
+        params
+    }
+    pub fn producer_parameters(&self) -> Vec<String> {
+        let mut params = Vec::new();
+        if let Some(producer) = self.producer.as_ref() {
+            if let Some(linger) = producer.linger {
+                params.push("--producer-linger".to_string());
+
+                params.push(format!("{}ms", linger.as_millis()));
+            }
+            if let Some(compression) = producer.compression {
+                params.push("--producer-compression".to_string());
+                params.push(format!("{}", compression.to_string()));
```

Comment:
Looks like we can to this:
```suggestion
                params.push(compression.to_string());
```

Reply:
Will apply this to see if we can merge this with this change